src/Sylius/Bundle/ShopBundle/templates/checkout/complete/form.html.twig

```diff
@@ -1 +1,15 @@
-{{ form_row(form.notes, sylius_test_form_attribute('extra-notes')|merge({'rows': 3})) }}
+{% set form = hookable_metadata.context.form %}
+{% set prefixes = hookable_metadata.prefixes|merge(['sylius_shop.shared.order.show']) %}
+
+{% form_theme form '@SyliusShop/Form/theme.html.twig' %}
+
+{{ form_start(form, {'action': path('sylius_shop_checkout_complete'), 'attr': {'novalidate': 'novalidate'}}) }}
+{{ form_errors(form) }}
+<input type="hidden" name="_method" value="PUT" />
+
+{% hook 'summary' with { _prefixes: prefixes } %}
+
+{% hook 'form' %}
```

Comment:
There should be only one hook per file. In this case,  should be a `form` hook, and inside it, a `summary` hook